api.ts

```diff
@@ -1151,6 +1153,33 @@ export interface SyncFormulaDef<
    * @hidden
    */
   updateOptions?: Pick<CommonPackFormulaDef<ParamDefsT>, 'extraOAuthScopes'>;
+
+  /**
+   * The javascript function that implements fetching permissions for a set of objects
+   * if the objects in this sync table have permissions in the external system.
+   *
+   * TODO(sam) pre merge:
+   * Does this need continuation? What if we fetch like 4 items and we need to continue for the last one?
+   * What are the size limits?
+   * Should we return as a flattened list or a map of lists?
+   * What do we do if there are no permissions on the objects but we just care about the permissions of the parent?
```

Comment:
This is part of a broader permissions inheritance issue. I think we likely can just add something new to Permissions that is like useParentPermissions. This works as long as the row contains an id that refers to the parent. Otherwise would need to flow in the parameters from the parent sync or something?